streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/SQLCacheExt.scala

```diff
@@ -42,6 +43,8 @@ class SQLCacheExt(override val uid: String) extends SQLAlg with WowParams {
 
     if (exe == "cache") {
       df.persist()
+      val viewNameAlias = params.get(viewName.name).get
+      SQLTableCacheClean.cache.put(viewName.name + "_" + viewNameAlias, viewNameAlias)
```

Comment:
Do not expose cache value in other package. Add a function to SQLTableCacheClean. E.g

```scala
SQLTableCacheClean.addCache(path,....)
```